src/frontend/config.ts

```diff
@@ -1,11 +1,101 @@
 import { createConfig, http } from 'wagmi'
-import { mainnet, polygon, sepolia } from 'wagmi/chains'
+import { Chain, hardhat, mainnet, polygon } from 'wagmi/chains'
+import { chainMap } from '@hyperplay/chains'
+
+type RpcUrls = {
+  http: string[]
+  webSocket?: string[]
+}
+
+type ChainMetadata = typeof chainMap[keyof typeof chainMap]
+
+export function parseChainMetadataToWagmiChain(
```

Comment:
shouldn't we move this to hp chains and export from there. making `wagmi/chains` a dev dependency?